randovania/data/json_data/prime1.txt

```diff
@@ -1279,379 +2367,822 @@ Asset id: 1125030300
 Elevator Access A
 Asset id: 639736833
-> Dock to Ore Processing; Heals? False
-  * Normal Door to Ore Processing/Dock to Elevator Access A
+> Dock to Ore Processing; Heals? False; Spawn Point
+  * Ice Door to Ore Processing/Dock to Elevator Access A
+  > Dock to Elevator A
+      Trivial
 
 > Dock to Elevator A; Heals? False
-  * Normal Door to Elevator A/Dock to Elevator Access A
+  * Ice Door to Elevator A/Dock to Elevator Access A
+  > Dock to Ore Processing
+      Morph Ball and Spider Ball
 
 ----------------
 Security Access B
 Asset id: 2718040532
 > Dock to Elite Research; Heals? False
-  * Normal Door to Elite Research/Dock to Security Access B
+  * Ice Door to Elite Research/Dock to Security Access B
+  > Dock to Mine Security Station
+      Trivial
 
-> Dock to Mine Security Station; Heals? False
-  * Normal Door to Mine Security Station/Dock to Security Access B
+> Dock to Mine Security Station; Heals? False; Spawn Point
+  * Wave Door to Mine Security Station/Dock to Security Access B
+  > Dock to Elite Research
+      Trivial
 
 ----------------
 Storage Depot A
 Asset id: 902158134
-> Dock to Mine Security Station; Heals? False
-  * Normal Door to Mine Security Station/Dock to Storage Depot A
+> Dock to Mine Security Station; Heals? False; Spawn Point
+  * Plasma Door to Mine Security Station/Dock to Storage Depot A
+  > Pickup (Flamethrower)
+      Trivial
+
+> Pickup (Flamethrower); Heals? False
+  * Pickup 76; Major Location? True
+  > Dock to Mine Security Station
+      Trivial
 
 ----------------
 Elite Research
 Asset id: 2325199700
 > Dock to Research Access; Heals? False
-  * Normal Door to Research Access/Dock to Elite Research
+  * Ice Door to Research Access/Dock to Elite Research
+  > Dock to Security Access B
+      After Rock Wall in Elite Research Destroyed
+  > Pickup (Missile)
+      Scan Visor and Morph Ball and Boost Ball and After Rock Wall in Elite Research Destroyed
 
-> Dock to Security Access B; Heals? False
-  * Normal Door to Security Access B/Dock to Elite Research
+> Dock to Security Access B; Heals? False; Spawn Point
+  * Ice Door to Security Access B/Dock to Elite Research
+  > Dock to Research Access
+      Scan Visor and Space Jump Boots and After Rock Wall in Elite Research Destroyed
+  > Event (Rock Wall in Elite Research Destroyed)
+      Scan Visor and Space Jump Boots and Morph Ball and Boost Ball
+  > Pickup (Missile)
+      Scan Visor and Space Jump Boots and Morph Ball and Boost Ball
+  > Pickup (Artifact of Warrior)
+      Power Bomb and Morph Ball and After Security Drone Defeated
+
+> Event (Rock Wall in Elite Research Destroyed); Heals? False
+  * Event Rock Wall in Elite Research Destroyed
+  > Dock to Research Access
+      Trivial
+
+> Pickup (Missile); Heals? False
+  * Pickup 78; Major Location? False
+  > Dock to Security Access B
+      Trivial
+
+> Pickup (Artifact of Warrior); Heals? False
+  * Pickup 77; Major Location? True
+  > Dock to Security Access B
+      Trivial
```

Comment:
- Add a helper node called "Top Floor". Connect it to the following nodes:
  - Event: Scan Visor, Morph Ball, Boost Ball, Bombs
  - Pickup (Missile): Scan Visor, Morph Ball, Boost Ball, Bombs, Space Jump
  - Security Access B: Trivial
  - Research Access: After Event
- Research Access should only connect to Top Floor with "After Event"
- Security Access B:
  - Remove connections to Research Access, Event, and Pickup (Missile)
  - Add connection to Top Floor with Space Jump and Scan Visor
- Pickup (Missile): Change connection to Top Floor instead of Security Access B with requirement as Space Jump